evaluate_material/src/parameterDB.cpp

```diff
@@ -0,0 +1,42 @@
+#include "parameterDB.h"
+
+#include "easi/Component.h"
```

Comment:
I don't know if that really important by <easi/Component.h>
(and other includes below).
I thought you would check them all.